translations/ko/intro/describing-quantum-computers.ipynb

```diff
@@ -0,0 +1,674 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "gloss": {
+          "complex-number": {
+            "text": "Complex numbers can be made from combinations of real and <a href='https://en.wikipedia.org/wiki/Imaginary_number'>imaginary numbers</a>. If you're not sure what this means, don't worry! You won't need to in this course (but you can <a href='https://en.wikipedia.org/wiki/Complex_number'>read more</a> if you're interested.",
+            "title": "Complex number"
+          },
+          "esoteric": {
+            "text": "Esoteric words are specialized words that most people won't understand. We'll try to avoid them as much as possible, but they can be useful sometimes.",
+            "title": "esoteric"
+          },
+          "real-number": {
+            "text": "A real number is a number that only has a magnitude. If you've not heard of a real number before, you can probably drop the 'real' and just call them 'numbers'. Examples of real numbers include: 112, -5.4, and π. <a href='https://en.wikipedia.org/wiki/Real_number'>Read more</a>.",
+            "title": "Real number"
+          }
```

Comment:
this notebook has some metadata (`gloss`) which should be translated (i.e., the values of `"text"` and `"title"`)

Reply:
> 
Hi @vabarbosa ! unfortunately, we cannot translate the meta through gitlocalize. I'm thinking about to translate almost every meta by myself at one point. then, is it OK with you approve this?

Reply:
@0sophy1 that is unfortunate, yes i can approve. i will investigate & try to find a way to make it easier to view/translate the info in the metadata